docs: task failure info (#5346)
aws/copilot-cli: site/content/blogs/release-v131.en.md

- **NLB enhancements**: You can now add security groups to Copilot-managed NLBs. NLBs also support the UDP protocol.
- **Better task failure logs**: Copilot will show more descriptive information during deployments when tasks fail, allowing better troubleshooting.

> You can view the ECS task stopped reasons for the two most recent tasks during deployments using `copilot [noun] deploy`.

Reviewer: lol we should add these section below https://github.com/aws/copilot-cli/pull/5346/files#diff-c933228c363a1118c64c52bd0c84346dbac20d74a73c370c3e0df06394003a10R59

Reviewer: i guess it only makes sense for ecs service deploy? `copilot [noun] deploy` might be a little confusing. Maybe we just say "... during deployments of your LBWS/BS/Worker svcs"? Also, it would be great to mention the reason why we wanted to prioritize this feature at the very beginning of the section (more context for readers)?

Author: > Also, it would be great to mention the reason why we wanted to prioritize this feature at the very beginning of the section.

what other info do you want me to add 🤔

Reviewer: Just some background will be good. Like without this feature what inconvenience they had before

> You can view the ECS task stopped reasons for the two most recent tasks during deployments using `copilot [noun] deploy`.

Reviewer 2: I think `copilot [noun] deploy` should be the subject of this sentence so you're using active voice.
```suggestion
`copilot [noun] deploy` will now display the ECS task stopped reasons for the two most recent task failures.
```

## Better task failure logs

> Before Copilot v1.31, in order to find out why your ECS tasks stopped, you'd have to navigate to AWS Console -> ECS -> Service -> Stopped Tasks -> Stopped Reason.

Reviewer 3: ```suggestion
Before Copilot v1.31, in order to find out why your ECS tasks stopped, you had to navigate to AWS Console -> ECS -> Service -> Stopped Tasks -> Stopped Reason.
```

Reviewer 3: or, alternatively:
```suggestion
Before Copilot v1.31, if you wanted to find out why your ECS tasks stopped, you'd have to navigate to AWS Console -> ECS -> Service -> Stopped Tasks -> Stopped Reason.
```

> Before Copilot v1.31, in order to find the ECS task stopped reason, You need to navigate to AWS Console -> ECS -> Service -> Stopped Tasks->Stopped Reason and find the reason for each task failure.

Reviewer 4: ```suggestion
Before Copilot v1.31, in order to find out why your ECS tasks stopped, you'd have to navigate to AWS Console -> ECS -> Service -> Stopped Tasks -> Stopped Reason.
```

>  With this enhancement, `copilot [noun] deploy` will now display the ECS task stopped reasons for the two most recent task failures during deployments of your Load Balanced Web Service, Backend Service and Worker Services.

Reviewer 4: ```suggestion
With this enhancement, `copilot [noun] deploy` will now display the ECS task stopped reasons within your CloudFormation deployment progress tracker. Copilot will show the two most recent task failures during deployments of your Load Balanced Web Service, Backend Service and Worker Services.
```

      TaskId    CurrentStatus   DesiredStatus                                                                                        
      6b1d6e32  DEPROVISIONING  STOPPED                                                                                              
      9802d212  STOPPED         STOPPED                                                                                              
                                                                                                                                     
    ✘ Latest 2 tasks stopped reason                                                                                 
      - [6b1d6e32,9802d212]: Essential container in task exited                                                                      
                                                                                                                                     
    Troubleshoot task stopped reason                                                                                         
      1. You can run `copilot svc logs --previous` to see the logs of the last stopped task.                                

>       2. You can visit this article https://repost.aws/knowledge-center/ecs-task-stopped.          

Reviewer 3: Sorry to add this nit so late in the process. 
```suggestion
      2. You can visit this article: https://repost.aws/knowledge-center/ecs-task-stopped.          
```
Can add that colon to the output?
